src/python/grpcio/grpc/_cython/_cygrpc/aio/common.pyx.pxi

```diff
@@ -67,3 +67,23 @@ class _EOF:
 
 
 EOF = _EOF()
+
+
+class BaseError(Exception):
+    """The base class for all exceptions generated by gRPC framework."""
+
+
+class UsageError(BaseError):
+    """Raised when the usage might lead to undefined behavior."""
+
+
+class AbortError(BaseError):
+    """Raised when calling abort in servicer methods.
+
+    This exception should not be suppressed. Applications may catch it to
+    perform certain clean-up logic, and then re-raise it.
+    """
+
+
+class InternalError(BaseError):
+    """Raised when unexpected error returned by Core."""
```

Comment:
How about "Raised upon unexpected errors in native code." "Core" is basically jargon. Basically no one outside of the gRPC team will understand it.

Reply:
Make sense! Updated to your version.